setup.py

```diff
@@ -98,7 +98,7 @@ def _get_dbt_core_version():
         ]
     },
     install_requires=[
-        "dbt-core~={}".format(dbt_core_version),
+        "dbt-core=={}".format(dbt_core_version),
```

Comment:
Why do you change this? I think you can use dbt-greenplum==1.4.0 with dbt==1.4.1

Reply:
dbt-postgres was failing installing 1.4.0 with core being 1.4.1 it wouldnt work without this change, ill try again and see if it will work without

Reply:
unable to reproduce the error i was having - probably environmental